dashboard/app/models/paired_user_level.rb

```diff
@@ -30,4 +30,34 @@ def self.pairs(user_level_ids)
       pluck(:navigator_user_level_id)
     return drivers | navigators
   end
+
+  # With just a pair (ha!) of queries, retrieve the set of paired UserLevel ids
+  # for a group of users of arbitrary size.
+  # @param [Enumerable<User>] users
+  # @return [Hash<Integer, Set<Integer>>] Map of User ids to sets of UserLevel ids
+  # Example response, given 1, 2, 3 are user ids and 101, 102, 103 are userlevel ids:
+  # {
+  #   1 => Set[101, 102],
+  #   2 => Set[102],
+  #   3 => Set[]
+  # }
```

Comment:
Thanks for including the examples in the comments - makes it very clear what's happening.